code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -3,6 +3,38 @@
 	name = "gear crate"
 	req_access = list(access_syndicate)
 
+/obj/structure/closet/crate/gear_loadout/hephaestus
+	desc = "A sturdy crate with Hephaestus Industries branding."
+	name = "hephaestus drop crate"
+	icon_state = "hephcrate"
+	icon_opened = "hephcrateopen"
+	icon_closed = "hephcrate"
+	layer = OBJ_LAYER - 0.01
```

Comment:
Is there a specific reason you added this all over the place?
And if so, why is it not set at one of the parent types?